modules/lx-autoscale/outputs.tf

```diff
@@ -2,30 +2,50 @@
 
 output "watchmaker-lx-autoscale-stack-id" {
   description = "CloudFormation stack id"
-  value       = "${aws_cloudformation_stack.watchmaker-lx-autoscale.id}"
+  value       = aws_cloudformation_stack.watchmaker-lx-autoscale.id
```

Comment:
We can simplify this whole thing now, using rich types for the output. Only need one output per _resource_ (and there's only the stack resource).

```
output "cloudformation_stack" {
  description = "CloudFormation stack object for watchmaker-lx-autoscale"
  value       = aws_cloudformation_stack.watchmaker-lx-autoscale
}
```

Reply:
This will require reworking examples and tests a bit also...